CHANGELOG.rst

```diff
@@ -1,5 +1,45 @@
 .. _changelog:
 
+0.25.7
```

Comment:
This should be `0.26.0`, we are adding a significant amount of features here, it's not just a patch release.